json-server-db.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Should this file also go inside tools folder?

Reply:
sure ill move it